.github/workflows/ci.yml

```diff
@@ -23,6 +23,8 @@ jobs:
     env:
       RAILS_ENV: test
       CC_TEST_REPORTER_ID: ${{ secrets.CC_TEST_REPORTER_ID }}
+      CI_NODE_TOTAL: ${{ secrets.CI_NODE_TOTAL || 1 }}
+      CI_NODE_INDEX: ${{ secrets.CI_NODE_INDEX || 0 }}
```

Comment:
I don't think this will be in secrets, it might be set with a matrix or something like that I guess